bigquery/indexer.py

```diff
@@ -82,17 +82,14 @@ def _get_nested_mappings(schema, prefix=None):
     return nested if nested else None
 
 
-def _create_nested_mappings(es, index_name, table_name, sample_id_col,
-                            billing_project_id):
+def _create_nested_mappings(es, index_name, table, sample_id_col):
     # Create nested mappings for repeated record type BigQuery fields so that
     # queries will work correctly, see:
     # https://www.elastic.co/guide/en/elasticsearch/reference/6.4/nested.html#_how_arrays_of_objects_are_flattened
-    project_id, dataset, table = table_name.split('.')
-    bq = bigquery.Client(project=billing_project_id)
-    t = bq.get_table(bq.dataset(dataset, project=project_id).table(table))
-    nested = _get_nested_mappings(t.schema, table_name)
+    table_name = table.full_table_id
```

Comment:
full_table_id is `bigquery-public-data:human_genome_variants.1000_genomes_sample_info`
table_name should be `bigquery-public-data.human_genome_variants.1000_genomes_sample_info`

Some background: BigQuery format used to be "project_id:dataset_id:table_name".
Legacy SQL uses this format (FROM project_id:dataset_id:table_name)
But then they switched to Standard SQL, which is "project_id.dataset_id:table_name".
We use Standard SQL format in our codebase. The BigQuery client libraries still use the legacy format.

This comes up several times in this PR. You may want to create _get_table_name(table) helper method.

As written, the mappings at http://localhost:9200/1000_genomes/_mappings?pretty are wrong. I'll add mappings to our integration tests in a future PR.

Reply:
Done.